### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by yash2121ja**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-pubsub/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

**Comment by yash2121ja** [comment]
update this

**Review by preyash2047 — APPROVED** [review]
Punctuation Correction

File: README.rst [hunk 1]
```diff
@@ -218,7 +218,7 @@ Community
 ---------
 
 Google Cloud Platform Python developers hang out in `Slack`_ in the ``#python``
-channel, click here to `get an invitation`_.
+the channel, click here to `get an invitation`_.
```
Comment (anguillanneuf) [inline README.rst:221]: nit: extra "the"

**Comment by anguillanneuf** [comment]
@yash2121ja Thanks for the fix in our readme. You still need to sign the CLA and update your commits to conventional commits. :)

**Comment by yash2121ja** [comment]
@googlebot I signed it!

**Comment by anguillanneuf** [comment]
@yash2121ja Also need to edit your commit message :-) Please make sure you update your branch too.

**Comment by anguillanneuf** [comment]
Close as no response is heard.

**Pull request closed without merging**